### ISSUE TITLE
Auto clear application data if the wrong pin is entered repeatedly

### ISSUE THREAD

**Issue by O35dE** [keepassium/KeePassium#356]
After a certain number of times the wrong PIN is entered, all local databases, backups and references to remote files are automatically deleted from the application.

This would add security to the use of the application and improve data protection in case of compromise of access to the iphone.

**Comment by keepassium** [comment]
Looks like a duplicate of #6

**Comment by O35dE** [comment]
Sry but no, they are completely different, suggestion in #6 is related to a special duress pin that you insert on purpose to delete everything, this suggestion here is similar to Strongbox’s approach - if someone inserts a given number of times the wrong pin, it will delete all local DBs, remote references, etc…

**Comment by keepassium** [comment]
You are right, duress PIN and several failed PIN attempts are different. Thank you for the thorough description. Reopening.

**Comment by keepassium** [comment]
Coming in the next update :)
